pkgs/top-level/all-packages.nix

```
@@ -27638,6 +27638,8 @@ with pkgs;
 
   ponymix = callPackage ../applications/audio/ponymix { };
 
+  popcorntime-bin = callPackage ../applications/video/popcorntime {};
```

```suggestion
  popcorntime = callPackage ../applications/video/popcorntime {};
```
also you are missing the package file.